src/lib_json/json_value.cpp

```diff
@@ -235,9 +235,7 @@ LogicError::LogicError(String const& msg) : Exception(msg) {}
 [[noreturn]] void throwRuntimeError(String const& msg) {
   throw RuntimeError(msg);
 }
-[[noreturn]] void throwLogicError(String const& msg) {
-  throw LogicError(msg);
-}
+[[noreturn]] void throwLogicError(String const& msg) { throw LogicError(msg); }
```

Comment:
clang-format did this. We should keep up with clang-format.

Reply:
Thanks, you are right.